docs/data_models/trace.md

```diff
@@ -33,9 +32,7 @@
 
 ## Trace Header
 
-```{eval-rst}
-.. autopydantic_model:: TraceHeaderDescriptor
-```
+Structured bla bla
```

Comment:
Did you want to leave this line in here?

Reply:
haha no, lemme fix it. thx.